### ISSUE TITLE
APP STORE BESTSELLER  LOOKS LIKE CHITCHAT

### ISSUE THREAD

**Issue by ericjosue** [stonesam92/ChitChat#104]
Hello guys, just wanted to raise an inquiry about this product  "Supertab for WhatsApp" (currently Best Seller in Mexico) that looks incredibly similar to ChitChat. I wonder if someone is making profit out of this project.

Kindly,

EJ

Link to the program:
https://itunes.apple.com/mx/app/supertab-for-whatsapp/id1050698659?l=en&mt=12

**Comment by ddulic** [comment]
That does seem like its the same app... Is there a way to report it without buying? I am new to the Mac and everything Apple.

**Comment by Rhaseven7h** [comment]
Although I'm a happy user of chitchat I have to mention that this is part
of your license:

"Permission is hereby granted, free of charge, to any person obtaining a
copy of this software and associated documentation files (the "Software"),
to deal in the Software without restriction, including without limitation
the rights to use, copy, modify, merge, publish, distribute, sublicense,
and/or sell copies of the Software, and to permit persons to whom the
Software is furnished to do so, subject to the following conditions:"

So, basically there's no infringement...




On Thursday, November 26, 2015, Damir Dulic <notifications@github.com>
wrote:

> That does seem like its the same app... Is there a way to report it
> without buying? I am new to the Mac and everything Apple.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stonesam92/ChitChat/issues/104#issuecomment-159899599>
> .
>


-- 


Gabriel Medina

**Comment by xylo21** [comment]
Although the 'developer' CrystalMinds (http://www.crystalminds.nl) is allowed sell it, its just not fair for consumers. I bought the app, have written an negative review and claimed my money back @ Apple :-)

**Comment by xylo21** [comment]
BTW, I see there are different apps using ChitChat. I have rated this app: https://itunes.apple.com/nl/app/berichten-voor-whatsapp/id1050148555?mt=12 

Its on no.1 paid in my Mac App Store.

**Comment by crystalminds** [comment]
That is not correct. Altho some apps definitely have ChitChat at it's core, such as the Supertab you mentioned, we did not do such a thing. We wrote the app ourselves from scratch and completely in Swift 2.0. We did not use ANY of the ChitChat components. It is of that reason we were able to enhance the experience for our users with added features like being the first to include Quick Reply in the notifications. Just because someone uses WhatsApp Web in a WebKit frame does not automatically mean they just copied everything from ChitChat. Please get your facts straight before making accusations like that. 

At time of writing we added GIF animations, Custom Wallpapers, Localizations, ability to determine where you want to run the app from (Dock or Menubar), we included a much better Unread Messages count in the badge and much more and much more to come.
